### PR TITLE
Update of documentation for Gemini-3f

### PR DESCRIPTION
**Pull request by extSunset**

### CODE REVIEW

File: docs/protocol/pulsar.mdx [hunk 1]
```diff
@@ -302,57 +297,58 @@ Ubuntu:
...
-Enter your node name to be identified on the network(defaults to `username`, press enter to use the default):
-Specify a path for storing plot files (press enter to use the default: `"/home/username/.local/share/subspace-cli/plots"`):
-Specify a path for storing node files (press enter to use the default: `"/home/username/.local/share/subspace-cli/node"`):
-Specify a plot size (defaults to `1000.0 MB`, press enter to use the default):
-Specify the chain to farm. Available options are: [Gemini3e, Dev, DevNet].
+Enter your node name to be identified on the network(defaults to `subspace`, press enter to use the default):
+Specify a path for storing plot files (press enter to use the default: `"/home/username/.local/share/pulsar/plots"`):
+Specify a path for storing node files (press enter to use the default: `"/home/username/.local/share/pulsar/node"`):
+Specify a plot size (defaults to `2.0 GB`, press enter to use the default):
+Specify the chain to farm. Available options are: [Gemini3f, Dev, DevNet].
```
Comment (EmilFattakhov) [inline docs/protocol/pulsar.mdx:326]: default version on the next line also needs to be changed Gemini3e -> Gemini3f

**Review by EmilFattakhov — COMMENTED** [review]
This looks great @extSunset, thanks a lot for updating our docs!

There is only one reference that needs to be changed, but other than that everything seems to work well. Nicely done!

File: docs/protocol/pulsar.mdx (hunk 1 above)
Comment (EmilFattakhov) [reply docs/protocol/pulsar.mdx:326]: Please feel free to do it directly here @extSunset!

Comment (EmilFattakhov) [reply docs/protocol/pulsar.mdx:326]: On the second though, let's keep this focused on Pulsar and address unrelated changes in separate PRs @extSunset. Thanks!

**Review by ImmaZoni — APPROVED** [review]
Looks great! 

Thanks for your submission @extSunset 🚀

**Pull request merged**